### DISCUSSION TITLE
Add support for backtick to use instead of double-quote at times for string-literal assignment

### DISCUSSION THREAD

**Discussion by merarischroeder** [dotnet/csharplang#3917] [General] [answered]
Solving this issue: https://stackoverflow.com/questions/14480724/escape-double-quotes-in-a-string

When there are a lot of double-quotes to escape within a string, it becomes labour-intensive and also less readable.

I propose that the backtick character can be used exactly the same as a double-quote character, except that escaping of double-quotes won't be necessary.

This is similar to how Javascript lets you use either single-quote or double-quote. It's nice to have both options.

The coder may then choose between backtick or double-quotes accordingly to define strings, and gain better readability when using.

``string test = `He said to me, "Hello World". How are you?`;``

Another alternative could be the MD way (which I just discovered while trying to make the code-block MD work) - see https://github.com/markdown-it/markdown-it/issues/604. If you 2x-double-quotes to start a string, that's how it ends, then you just use of a 1x-double-quote becomes literal.

`string test = ""He said to me, "Hello World". How are you?"";`

**Comment by HaloFour** [comment, marked as answer]
See also: https://github.com/dotnet/csharplang/discussions/89
